src/librustc_error_codes/error_codes/E0749.md

```diff
@@ -1,4 +1,19 @@
-Negative impls are not allowed to have any items. Negative impls
-declare that a trait is **not** implemented (and never will be) and
-hence there is no need to specify the values for trait methods or
-other items.
+An item was added on a negative impl.
+
+Erroneous code example:
+
+```compile_fail,E0749
+# #![feature(negative_impls)]
+trait MyTrait {
+    type Foo;
+}
+
+impl !MyTrait for u32 {
+    type Foo = i32; // error!
+}
+# fn main() {}
```

Comment:
Do we suggested how to fix this? Is it clear that the user should remove the type?

Reply:
Hm it's verbose for me. "item" here is ambiguous a bit?

Reply:
The explanation below isn't enough?

Reply:
Just enough but it is not as straightforward.

Reply:
Hum... I'm not sure to see what's missing so could please open a PR with your vision? (let's switch position for once! :p)

Set me as reviewer on it so we can move forward quickly! :)